packages/samples/common-types/gen.ts

```diff
@@ -86,6 +86,7 @@ function cleanupDocument(original: OpenAPI2Document): OpenAPI2Document {
 
   replaceUuidRefs(document, "Azure.Core.uuid");
   replaceUuidRefs(document, "Azure.Core.azureLocation");
+  replaceUuidRefs(document, "Azure.Core.armResourceType");
```

Comment:
What does this actually do? It seems like this should have some impact on a generated common-types swagger, but I don't see it in this PR

Reply:
this line is added to keep this file is unchanged.
I followed the same pattern for azurelocation and uuid above.
Without this, the generated swagger will look like this:
"property": {
"$ref": "#/definitions/armResourceType"
},
// in the definitions
"armResourceType": {
"type": "string"
}

Reply:
so, this method should just be named 'omitScalarSchema` :-)

Reply:
LOL indeed - so this method was first used on uuid and later we found we should apply that to more types but we did not rename the method.